test/solidlsp/clojure/test_clojure_basic.py

```diff
@@ -0,0 +1,230 @@
+from pathlib import Path
```

Comment:
Could you pls also add at least one test case for closure to test/test_serena_agent.py? I found it valuable to also test language server implementations using the high-level interfaces, even though the test logic is largely overlapping